runtime/services/multimediad/service.js

```diff
@@ -98,6 +105,11 @@ MultiMedia.prototype.resume = function (appId) {
     if (this.handle[appId] && !this.handle[appId].playing) {
       this.handle[appId].resume()
       AudioManager.setPlayingState(audioModuleName, true)
+
+      // need to send events only when the state is switched
+      process.nextTick(() => {
```

Comment:
Why schedule this emit on next tick?

Reply:
In order to get the dbus connection back immediately. Because pause will send an event via dbus.